docs/units/combining_and_defining.rst

```diff
@@ -98,4 +100,4 @@ can be enabled by calling :func:`~astropy.units.add_enabled_units`::
     kmph         | 0.277778 m / s  |         ,
   ]
 
-.. EXAMPLE END
+.. EXAMPLE END
```

Comment:
Why is this line marked as having changed?

Reply:
I suppose that as I am adding a new line for the link, there is one new line added in the end of the file and that's what is being marked.